.travis.yml

```diff
@@ -6,16 +6,21 @@ notifications:
 matrix:
   fast_finish: true
   include:
-    - python: 2.7
-      env: CONDA_ENV=py27
-    - python: 2.7
-      env: CONDA_ENV=py27-min
-    - python: 3.4
-      env: CONDA_ENV=py34
-    - python: 3.5
-      env: CONDA_ENV=py35
-    - python: 3.6
-      env: CONDA_ENV=py36
+  - python: 2.7
```

Comment:
Please revert the indentation to what it was before.  Unlike Python, YAML doesn't care much about whitespace, so this is just stylistic.  But I like the original indentation we had, and -- more importantly -- it's consistent with the indentation in the rest of the fall and in all of our other .yml files.

Reply:
the old indentation throws an error `syntax error: (<unknown>): did not find expected '-' indicator while parsing a block collection at line 9 column 5` in the travis build

Reply:
Sorry, you're totally right!  Should have checked the Travis log from your first commit before acting like I knew what I was talking about.

So ignore that comment.

Reply:
For what it's worth, the following checks out on http://lint.travis-ci.org/ (note I've de-indented `allow_failures:` by two spaces, but it's too late now :)

```
language: python
sudo: false
notifications:
  email: false

matrix:
  fast_finish: true
  include:
    - python: 2.7
      env: CONDA_ENV=py27
    - python: 2.7
      env: CONDA_ENV=py27-min
    - python: 3.4
      env: CONDA_ENV=py34
    - python: 3.5
      env: CONDA_ENV=py35
    - python: 3.6
      env: CONDA_ENV=py36
    - python: 3.6
      env: CONDA_ENV=py36-xarray-dev
  allow_failures:
    - python: 3.6
      env: CONDA_ENV=py36-xarray-dev

before_install:
  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
      wget https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh -O miniconda.sh;
    else
      wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
    fi
  - bash miniconda.sh -b -p $HOME/miniconda
  - export PATH="$HOME/miniconda/bin:$PATH"
  - hash -r
  - conda config --set always_yes yes --set changeps1 no
  - conda update -q conda
  - conda info -a

install:
  - conda env create --file ci/environment-$CONDA_ENV.yml
  - source activate test_env
  - pip install -e .

script:
  - py.test aospy --cov=aospy --cov-report term-missing
after_success:
  - coveralls
```

Reply:
Ah good catch!  I prefer this.  Will switch to this at some point.